decidim-consultations/db/migrate/20180115170933_add_slug_to_decidim_consultations_questions.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class AddSlugToDecidimConsultationsQuestions < ActiveRecord::Migration[5.1]
+  def change
+    add_column :decidim_consultations_questions,
+               :decidim_organization_id,
+               :integer,
+               index: {
+                 name: "index_decidim_questions_on_decidim_organization_id"
+               }
+
+    add_column :decidim_consultations_questions, :slug, :string
+
+    Decidim::Consultations::Question.find_each do |question|
```

Comment:
Instead of using the real model, use plain SQL, or define a dummy model in order to access the underlying table like this:

```ruby
class AddSlugToDecidimConsultationsQuestions < ActiveRecord::Migration[5.1]
  class Question < ApplicationRecord
    self.table_name = :decidim_consultations_questions
  end
```

and then use `Question`, instead of `Decidim::Consultations::Question`. That way you make sure you don't load the real model during migrations.